vendor/assets/javascripts/jquery.lazyload.js

```diff
@@ -0,0 +1,254 @@
+/*!
```

Comment:
Ah! Is this something we could include via `yarn` instead, pegged to a specific version? Thank you!!!

Reply:
Yeah done! I still have to add more routes and refine this so no review required right now, I will ping when done :sweat_smile: